packages/E2ETest/wdio/test/DirectManipulation.spec.ts

```diff
@@ -8,7 +8,7 @@ import DirectManipulationPage from '../pages/DirectManipulationPage';
 import assert from 'assert';
 
 beforeAll(() => {
-  HomePage.backToHomePage();
+  //  HomePage.backToHomePage();
```

Comment:
Could we leave a comment about why this is commented out?

Reply:
Thanks for the reminder.  This code will never be necessary so I just removed it from all the tests.

---
In reply to: [457675832](https://github.com/microsoft/react-native-windows/pull/5555#discussion_r457675832) [](ancestors = 457675832)